modules/admin-ui-index/src/main/java/org/opencastproject/adminui/index/AdminUISearchIndex.java

```diff
@@ -22,13 +22,13 @@
 package org.opencastproject.adminui.index;
 
 import org.opencastproject.index.service.api.EventIndex;
-import org.opencastproject.index.service.impl.index.AbstractSearchIndex;
-import org.opencastproject.index.service.impl.index.event.Event;
-import org.opencastproject.index.service.impl.index.event.EventIndexSchema;
-import org.opencastproject.index.service.impl.index.group.Group;
-import org.opencastproject.index.service.impl.index.series.Series;
-import org.opencastproject.index.service.impl.index.theme.Theme;
-import org.opencastproject.index.service.impl.index.theme.ThemeIndexSchema;
+import org.opencastproject.search.index.AbstractSearchIndex;
+import org.opencastproject.search.index.event.Event;
+import org.opencastproject.search.index.event.EventIndexSchema;
+import org.opencastproject.search.index.group.Group;
+import org.opencastproject.search.index.series.Series;
+import org.opencastproject.search.index.theme.Theme;
+import org.opencastproject.search.index.theme.ThemeIndexSchema;
```

Comment:
These now use the same namespace as the completely unrelated search service.
That's confusing. Can we use something else?

https://github.com/opencast/opencast/blob/e9f79c2d585e3c5facc24d04681753b948704041/modules/search-service-api/src/main/java/org/opencastproject/search/api/SearchService.java#L23

Reply:
Good point. I'm gonna try to rename this into something else, probably "elasticsearch".

Reply:
I renamed this module and its namespace and also the modules search and search-api, which had the same problem, from `search` to `elasticsearch` (and added an `-impl` in one case). Things shouldn't be quite so confusing now. 

I didn't touch class names for now, otherwise I'd still be here next week. The same goes for modules with `index` in their name, since then I'd have to move the index service, and I don't want to deal with that right now. :D